lib/sockunion.c

```diff
@@ -587,13 +587,10 @@ static void __attribute__((unused)) sockunion_print(const union sockunion *su)
 
 	switch (su->sa.sa_family) {
 	case AF_INET:
-		printf("%s\n", inet_ntoa(su->sin.sin_addr));
+		printf("%pI4\n", &su->sin.sin_addr);
 		break;
 	case AF_INET6: {
-		char buf[SU_ADDRSTRLEN];
-
-		printf("%s\n", inet_ntop(AF_INET6, &(su->sin6.sin6_addr), buf,
-					 sizeof(buf)));
+		printf("%pI6\n", &su->sin6.sin6_addr);
 	} break;
```

Comment:
Can you please remove the curly brackets too?

Reply:
+1

Reply:
fixed